src/ui/display.c

```diff
@@ -253,6 +253,11 @@ UX_STEP_NOCB(ux_display_warning_external_inputs_step,
                  "external inputs",
              });
 
+// Step with warning icon for unverified inputs (segwit inputs with no non-witness-utxo)
+UX_STEP_NOCB(ux_unverified_segwit_input_flow_1_step, pb, {&C_icon_warning, "Unverified inputs"});
+UX_STEP_NOCB(ux_unverified_segwit_input_flow_2_step, nn, {"Update", " Ledger Live"});
```

Comment:
Do you need the space before Ledger live ?

Reply:
Removed, thanks.